dist/index.html

```diff
@@ -29,8 +29,25 @@
       Activites
     </section>
 
-    <section class="donation-banner">
-      Donation Banner
+    <section class="donation-banner bg-c100 relative font-body">
+      <div class="donation-bnanner__icon bg-c000 rounded-full absolute"></div>
```

Comment:
> I have attached the Icon so that you complete your stunning section 👍 
![hope-icon](https://user-images.githubusercontent.com/8650435/79672623-e3a52900-81e4-11ea-9746-68abc85bac6d.png)